assets/Product.wxs

```diff
@@ -60,17 +60,27 @@
 
     <!--We need to show EULA, and provide option to customize download location-->        
     <Property Id="WIXUI_INSTALLDIR" Value="INSTALLFOLDER" />
-	
-    <!-- Prerequisites check for Windows Universal C time and Visual Studio 2015 C++ redistributables -->
+
+    <!-- Prerequisite check for Windows Universal C time -->
     <Property Id="UCRTINSTALLED" Secure="yes">
-      <DirectorySearch Id="Windows_System32" Path="[WindowsFolder]System32" Depth="0">
+      <DirectorySearch Id="Windows_System32_search1" Path="[WindowsFolder]System32" Depth="0">
```

Comment:
Would it be better to call this `Universal_Runtime_Search`?

Reply:
I can make 'search' uppercase but the numbers at the end are needed now because otherwise WiX gives a compilation error that the DirectorySearch Id is duplicated. This seems to be a WiX peculiarity in the case of file search properties and I have seen other people do the same as well but if you know a better way of doing it, feel free to propose.

Reply:
I see.  Not a blocking issue for me.

Reply:
Sorry, I see only now that I misread your 2 comment value suggestions as `Windows_System32_Search`, which would have resulted in the duplicated DirectorySearch Id compilation error (which I actually had in the first place when I just copy pasted the existing block as a template with some modifications). So, yes, you could name it `Universal_Runtime_Search` but I am not sure if there is much value because the property name already contains this information and this Id is not used anywhere (but needed for compilation). Maybe it is better to rename the property value(s) instead?

Reply:
My thinking is that if we need to extend this in the future, `Windows_System32_searchN` doesn't really provide much information.  If the requirement is simply it has to be unique, I think something more descriptive to what the search does is preferred.

Reply:
OK, I did more research and found a documented solution, which allows to define it once and then reference it in order to get rid of this problem.